workflows/rnaseq.nf

```diff
@@ -205,30 +187,37 @@ workflow RNASEQ {
     }
 
     //
-    // SUBWORKFLOW: Read in samplesheet, validate and stage input files
+    // Create input channel from input file provided through params.input
     //
-    INPUT_CHECK (
-        file(params.input)
-    )
-    .reads
-    .map {
-        meta, fastq ->
-            new_id = meta.id - ~/_T\d+/
-            [ meta + [id: new_id], fastq ]
-    }
-    .groupTuple()
-    .branch {
-        meta, fastq ->
-            single  : fastq.size() == 1
-                return [ meta, fastq.flatten() ]
-            multiple: fastq.size() > 1
-                return [ meta, fastq.flatten() ]
-    }
-    .set { ch_fastq }
-    ch_versions = ch_versions.mix(INPUT_CHECK.out.versions)
-    // TODO: OPTIONAL, you can use nf-validation plugin to create an input channel from the samplesheet with Channel.fromSamplesheet("input")
-    // See the documentation https://nextflow-io.github.io/nf-validation/samplesheets/fromSamplesheet/
-    // ! There is currently no tooling to help you write a sample sheet schema
+    Channel
+        .fromSamplesheet("input")
+        .map {
+            meta, fastq_1, fastq_2 ->
+                if (!fastq_1.exists()) {
+                    error("Please check input samplesheet -> Read 1 FastQ file does not exist!\n${fastq_1}")
+                }
+                if (fastq_2) {
+                    if(!fastq_2.exists()) {
+                        error("Please check input samplesheet -> Read 2 FastQ file does not exist!\n${fastq_2}")
+                    }
```

Comment:
These `exists` checks are already performed by the plugin so these will never trigger

Reply:
Yep, this is one of the things I am going to remove. There is a bug in the plugin. See [here](https://nfcore.slack.com/archives/C056RQB10LU/p1688644662600359?thread_ts=1688633898.260909&cid=C056RQB10LU)

Reply:
Perfect :p Then this is a perfect example on how to implement the plugin :tada:

Reply:
Fixed in https://github.com/nf-core/rnaseq/pull/1058/commits/3c5fa30c16436fd3e5cbc7655612b25f26e62f49